docs/examples/CollapsablePanel.js

```diff
@@ -0,0 +1,72 @@
+var CollapsablePanel = React.createClass({
```

Comment:
That was the intent  - a simplified version of the existing `Panel`.  I can change the name if you think that would be confusing.